src/inference/src/core.cpp

```diff
@@ -156,7 +157,7 @@ CompiledModel Core::compile_model(const std::shared_ptr<const ov::Model>& model,
                                   const RemoteContext& context,
                                   const AnyMap& config) {
     OV_CORE_CALL_STATEMENT({
-        auto exec = _impl->compile_model(model, context, config);
+        auto exec = _impl->compile_model(model, ov::SoPtr<ov::IRemoteContext>{context._impl, context._so}, config);
```

Comment:
is it possible to write 
```suggestion
        auto exec = _impl->compile_model(model, context._impl, context._so, config);
```
?